ruby_04-apis_and_scalability/js_refactor_tractor.md

```diff
@@ -303,6 +303,141 @@ __Discussion Points__
 
 #### Dead Code Among the Living
 
+#### If If If Else If Else Else If Else ...
+
+Deeply nested if/else statements should be avoided-- they make a program difficult to follow. There are many ways to refactor/restructure to avoid this condition, and good strategy can vary with the situation.
+
+##### 1. Break up into functions.
```

Comment:
Might want to make these `######`, as they are currently the same header size as the main 'If Else...'